### PR TITLE
Api suppliers endpoint documentation

### PR DESCRIPTION
**Pull request by Psilolouben**
Addresses [this task](https://phabricator.skroutz.gr/T119650)

### CODE REVIEW

File: source/localizable/smart_cart/_fbs_supplier_object.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+`email`      | String | Supplier Email
+
+#### Supplier creation/update options
+
+These are valid values to POST to the corresponding keys when creating or updating fbs suppliers.
+
+Name                              | Type   | Example     | Description
+--------------------------------- | ------ | ----------- | --------------------------------------------------
+`new_supplier.id`                 | String | "abs30220"              | The supplier ID. Can be fetched from the [list of the shop's suppliers](/smart_cart/fbs_api/#retrieve-shop-suppliers)
+`new_supplier.code`               | String | "a_code"                | Supplier Code
```
Comment (tassosMaurides) [inline source/localizable/smart_cart/_fbs_supplier_object.html.md.erb:23]: Not exactly part of this PR but supplier code should be unique per shop for SARMED. 
Will we handle this constraint in our systems or add a validation and require unique supplier-code values from merchants?

Maybe we should address this issue before making the documentation of the API public

Comment (Psilolouben) [reply source/localizable/smart_cart/_fbs_supplier_object.html.md.erb:23]: Created a [task](https://phabricator.skroutz.gr/T120897) for this. Will update documentation here accordingly.

Comment (Psilolouben) [reply source/localizable/smart_cart/_fbs_supplier_object.html.md.erb]: [fixed](https://github.com/skroutz/developer.skroutz.gr/pull/196/commits/073c46589f1364844a1732b56adefb99436da3d4)

**Review by tassosMaurides — APPROVED** [review]
:+1: Nice

**Pull request merged**